tests/util.py

```diff
@@ -23,7 +23,7 @@
 from sphinx.ext.autodoc import AutoDirective
 from sphinx.pycode import ModuleAnalyzer
 
-from path import path
+from path import path, repr_as
```

Comment:
Well, this *is* used for exporting from this module.  (It doesn't *live* in `util.py` because of the import hierarchy.)